packages/local_auth/CHANGELOG.md

```diff
@@ -61,10 +72,12 @@
 
 * Replace deprecated `getFlutterEngine` call on Android.
 
+
```

Comment:
Remove these extra newlines. It might be due to formatting done in your editor?

Reply:
extra newlines removed